bridge/client/app/_models/secret.ts

```diff
@@ -1,3 +1,9 @@
+export enum Scopes {
+  KeptnDefault = 'keptn-default',
```

Comment:
Just a remark ... I added it also in my PR here:  `shared\interfaces\secret.ts`
as: 
```
export enum SecretScope {
  DEFAULT = 'keptn-default',
  WEBHOOK = 'keptn-webhook-service'
}
```
This is needed to retrieve secrets only for a specific scope, so as server and bridge make use of it, can we streamline it?

Reply:
Also I would vote for enum name `SecretScope` as there could be other scopes too.